cms/djangoapps/contentstore/views/course.py

```diff
@@ -439,7 +439,12 @@ def course_filter(course_summary):
 
         return has_studio_read_access(request.user, course_summary.id)
     if org is not None:
-        courses_summary = [] if org == '' else CourseOverview.get_all_courses(orgs=[org])
+        orgs = []
+        if isinstance(org, list):
+            orgs.extend(org)
+        else:
+            orgs.append(org)
```

Comment:
we can also do it like this:
```
if not isinstance(org, list):
    orgs.append(org)
```

Reply:
@taimoor-ahmed-1 sine we have to append or extend based on the type so the current implementation seems more readable to me.

@rehan99000 Please add an extra line after de-indenting the code

Reply:
@zubair-arbi done

@taimoor-ahmed-1 I have done it for more readability and backward compatibility.